appium-dotnet-driver/Appium/PageObjects/ByFactory.cs

```diff
@@ -108,7 +108,7 @@ private static List<By> GetListOfComplexBys(bool useSequence, bool useAll, List<
                 return null;
 
             Attribute[] attributes = null;
-            if (upperPlatform.Equals(MobilePlatform.Android.ToUpper()) & (upperAutomation != null &&
+            if (upperPlatform.Equals(MobilePlatform.Android.ToUpper()) && (upperAutomation != null &&
```

Comment:
There is a need to check both the conditions here that's why @TikhomirovSergey has put `&` over `&&`.